### DISCUSSION TITLE
My Ideas

### DISCUSSION THREAD

**Discussion by UnderboladoTB** [BedrockTweaks/Files#384] [Ideas]
1. Minecraft Legends and Story Mode Textures
2. --Orange--Birch--Leaves-- Edit Someone alr suggested it
3. Pigman Texture for Villagers (or for Zombie Pigmans)
4. Rainbow Wool / Rainbow Sheep
5. More Mob Heads
6. 3D Plants
7. 3D Iron Bars, Sugar Cane, Ladders and Vines
8. 4J Logos and Stuff
9. More Villager Variants
10. Pigs with Crepper Texture and Crepper with Pig Texture
11. Make Giants, Killer Rabbits and Zombie Horses Spawn Naturally
12. Boat's Recipe needs Shovel
13. XP Bottles have a Recipe
14. Invisible Armor
15. Bundles needs Rabbit Hide instead of Leather for crafting it

**Comment by SquatchHunter** [comment]
- 3d packs are not possible, 
- more mob heads is planned with the addons update, 
- XP bottles I have already made an addon for (waiting for addon update)
the rest is all "community packs" and not a high priority.

**Reply by UnderboladoTB** [reply]
Alr
